app/models/repository.rb

```diff
@@ -26,6 +26,20 @@ def self.find_lite(id, structural: true)
     NilModel.new(id)
   end
 
+  # @param [String] id the identifier for the item to be found
+  # @param [String] user_version the user version to be found
+  # @return [Cocina::Models::DRO,Cocina::Models::Collection,Cocina::Models::AdminPolicy,NilModel] cocina model instance corresponding to the given druid
+  # @raise [Dor::Services::Client::NotFoundResponse] when dor-services-app cannot find the requested user version
+  def self.find_user_version(id, user_version)
+    raise ArgumentError, 'Missing identifier' unless id
+    raise ArgumentError, 'Missing user_version' unless user_version
+
+    object_client = Dor::Services::Client.object(id)
+    object_client.user_version.find(user_version)
+  rescue Dor::Services::Client::UnexpectedResponse
+    NilModel.new(id)
```

Comment:
Also shows as uncovered.

Reply:
So I think NilModel is vestigial, but I'm not entirely sure. I'm going to add an HB alert to help determine.

Reply:
Great idea.